apps/fyler/src/fyler_category_manager.erl

```diff
@@ -410,6 +416,7 @@ tasks_({task_rejected, Ref, Node}, #state{category = Category, retasks = ReTasks
         undefined ->
           {next_state, tasks, State#state{pools = NewPools}};
         Task ->
+          ?D({task_rejected, Id}),
           {NextState, NewState} = choose_pool(State#state{retasks = fyler_queue:in(Task, ReTasks), pools = NewPools}),
```

Comment:
Line 420 is too long:           {NextState, NewState} = choose_pool(State#state{retasks = fyler_queue:in(Task, ReTasks), pools = NewPools}),.